politeiawww/www.go

```diff
@@ -788,6 +829,8 @@ func _main() error {
 		permissionPublic, false)
 	p.addRoute(http.MethodGet, v1.RouteCommentsGet, p.handleCommentsGet,
 		permissionPublic, true)
+	p.addRoute(http.MethodGet, v1.RouteUserProposals, p.handleUserProposals,
```

Comment:
Are we sure we want this to be unprivileged?

Reply:
I don't think there's any issue to showing a user's public proposals to users who aren't logged in; they can already see the proposals themselves. If you think the users should be hidden from the public UI, though, we could do that and make this route protected by login.